### PR TITLE
SLM-257 Accessibility audit medium priority issues

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: server/filters/renderChooseBarcodeOptionRadiosFilter.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function renderChooseBarcodeOptionRadiosFilter(values: Array<stri
         html: `        
           <img src='/assets/images/print-barcodes-image.png' alt='' width='94px'/>
           <details class='govuk-details' data-module='govuk-details'>
-            <summary class='govuk-details__summary'><span class='govuk-details__summary-text'>How it works</span></summary>            
+            <summary class='govuk-details__summary'><span class='govuk-details__summary-text'>How it works <span class='govuk-visually-hidden'> (printing as part of your own document)</span></span></summary>
```
Comment (mikehalmamoj) [inline server/filters/renderChooseBarcodeOptionRadiosFilter.ts:12]: When using a screen reader users can receive a summary of clickable components. In this summary it is not possible to differentiate between the two "How it works" links, so more context was required for each link.

File: package.json [hunk 2]
```diff
@@ -114,7 +114,6 @@
     "jsonwebtoken": "^8.5.1",
     "jwt-decode": "^3.1.2",
     "moment": "^2.29.3",
-    "multer": "^1.4.4",
```
Comment (mikehalmamoj) [inline package.json:117]: Should have been removed when we remove the multipart form upload capability. Just noticed it because it has a CVE.

**Review by hyderkhanmoj — APPROVED** [review]
LGTM

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Pull request merged**